packages/samples/react/src/components/SampleDescription.tsx

```diff
@@ -18,10 +18,18 @@ export const SampleDescription: FC<PropsWithChildren> = (props) => {
 			: `${PUBLIC_DOC_COMPONENT_URL}/${componentName}`;
 	}, [location.href]);
 
+	const codeLink = useMemo(() => {
```

Comment:
Scenario funktioniert noch nicht, würde aber mit  `location.hash`  wohl funktionieren

Reply:
Ich verstehe gerade noch nicht, warum das mit `location.hash` funktionieren würde.
Das Problem, das ich sehe, ist in der Benennung der Scenario Source-Files: Hier wurde PascalCase statt (wie bei den anderen Samples und in den Routes) kebab-case verwendet. Ich denke das können bzw. sollten wir anpassen.

Reply:
Ich nutze stets Kebab-Case noch aus Angular 1 Zeiten. Das würde ich befürworten.